### PR TITLE
[Feature] Add statistics related show statement

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] bug
- [x] feature
- [ ] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

### Added user interface
- View analyze history: show analyze status
- View the analysis status of the table, record the type and time of the last successful analysis: show stats meta/ show histogram meta
- View the list of automatically collected tasks: show analyze job, the meaning is the same as the original show analyze

### CODE REVIEW

**Review by Seaven — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:disappointed: **fail** : 237 / 421 (56.29%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/http/rest/QueryDumpAction.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/service/FrontendServiceImpl.java|0|5|00.00%|
|:large_blue_circle:|com/starrocks/load/loadv2/SparkLoadJob.java|0|3|00.00%|
|:large_blue_circle:|com/starrocks/statistic/StatisticAutoCollector.java|0|2|00.00%|
|:large_blue_circle:|com/starrocks/sql/ast/CreateAnalyzeJobStmt.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/sql/parser/SqlParser.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/load/loadv2/BrokerLoadJob.java|0|3|00.00%|
|:large_blue_circle:|com/starrocks/qe/StmtExecutor.java|0|7|00.00%|
|:large_blue_circle:|com/starrocks/persist/EditLog.java|0|15|00.00%|
|:large_blue_circle:|com/starrocks/journal/JournalEntity.java|0|9|00.00%|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeManager.java|3|49|06.12%|
|:large_blue_circle:|com/starrocks/qe/ShowExecutor.java|4|40|10.00%|
|:large_blue_circle:|com/starrocks/statistic/BasicStatsMeta.java|18|30|60.00%|
|:large_blue_circle:|com/starrocks/statistic/HistogramStatsMeta.java|14|20|70.00%|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeJob.java|10|14|71.43%|
|:large_blue_circle:|com/starrocks/sql/ast/ShowAnalyzeJobStmt.java|36|47|76.60%|
|:large_blue_circle:|com/starrocks/sql/ast/ShowAnalyzeStatusStmt.java|37|44|84.09%|
|:large_blue_circle:|com/starrocks/statistic/AnalyzeStatus.java|27|32|84.38%|
|:large_blue_circle:|com/starrocks/sql/ast/ShowBasicStatsMetaStmt.java|24|28|85.71%|
|:large_blue_circle:|com/starrocks/sql/ast/ShowHistogramStatsMetaStmt.java|24|28|85.71%|
|:large_blue_circle:|com/starrocks/sql/analyzer/AnalyzeStmtAnalyzer.java|24|26|92.31%|
|:large_blue_circle:|com/starrocks/sql/parser/AstBuilder.java|8|8|100.00%|
|:large_blue_circle:|com/starrocks/sql/ast/AnalyzeHistogramDesc.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/common/Config.java|5|5|100.00%|
|:large_blue_circle:|com/starrocks/statistic/Constants.java|2|2|100.00%|

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java [hunk 1]
```diff
@@ -1026,6 +1029,10 @@ private boolean loadTxnCommitImpl(TLoadTxnCommitRequest request) throws UserExce
                 entity.counterRoutineLoadFinishedTotal.increase(1L);
                 entity.counterRoutineLoadBytesTotal.increase(routineAttachment.getReceivedBytes());
                 entity.counterRoutineLoadRowsTotal.increase(routineAttachment.getLoadedRows());
+                if (basicStatsMeta != null) {
+                    basicStatsMeta.increase(routineAttachment.getLoadedRows());
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/service/FrontendServiceImpl.java:1033]: you could update loaded rows in OlapTableTxnLogApplier.applyCommitLog, or DatabaseTransactionMgr.updateCatalogAfterCommitted  and only need do once.

**Review by Youngwb — APPROVED** [review]

**Pull request merged**